test/mocking_classes.py

```diff
@@ -140,10 +142,13 @@ def __init__(self, device, batch_size=None):
         self.reward_spec = NdUnboundedContinuousTensorSpec((1,))
         self.counter = 0
 
-        self.set_seed = MockSerialEnv.set_seed
+    set_seed = MockSerialEnv.set_seed
+    rand_step = MockSerialEnv.rand_step
 
     def _step(self, tensordict):
         self.counter += 1
+        if self.batch_locked:
```

Comment:
This is actually done inside the step of batch env, so no need to redo it

Reply:
Oh ok!